Sources/Plasma/PubUtilLib/plPhysX/CMakeLists.txt

```diff
@@ -36,7 +36,17 @@ set(plPhysX_HEADERS
 )
 
 add_library(plPhysX STATIC ${plPhysX_SOURCES} ${plPhysX_HEADERS})
-target_link_libraries(plPhysX ${PHYSX_LIBRARIES})
+
+target_link_libraries(plPhysX
+    PUBLIC
+        ${PHYSX_LIBRARIES}
+        CoreLib
+        plPhysical
+)
+
+if(UNIX)
+    target_link_libraries(plPhysX PUBLIC ${CMAKE_DL_LIBS})
+endif(UNIX)
```

Comment:
If PhysX itself requires ldl, then it should be added to PHYSX_LIBRARIES in the find module.